README.md

```diff
@@ -130,13 +138,20 @@ OPTIONS
             Only take logs for jobs started N days or hours ago: Nd for N days
             and Nh for N hours.
 
+    --repo-path
+    
+            repository (without owner)
+            Uses in `fetch.py` result files path, so `gather_data.py` needs it
+            to find data for certain repo. Default: 'tarantool/tarantool'.
+            You can set only one repo in one sckript start.
+
 EXAMPLE
     
     Collect data about jobs and tests started a week ago or later, and put
-    this data to InfluxDB:
+    this data to InfluxDB in repo 'tarantool/multivac:
```

Comment:
repo → bucket?

Reply:
No. Repo -> path to directory with fetched workflow run jobs and workflow runs